features/ownership/get-creator.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Test getting creator ownership
+    Background:
+        Given I am using the UDB3 base URL
+        And I am using an UiTID v1 API key of consumer "uitdatabank"
+        And I send and accept "application/json"
+
+    Scenario: Getting the creator of an ownership as creator
+        Given I am authorized as JWT provider v2 user "dev_e2e_test"
+        And I create a minimal organizer and save the "id" as "organizerId"
+        And I request ownership for "d759fd36-fb28-4fe3-8ec6-b4aaf990371d" on the organizer with organizerId "%{organizerId}" and save the "id" as "ownershipId"
+        When I send a GET request to "/ownerships/%{ownershipId}/creator"
+        Then the response status should be 200
+        And the JSON response at "userId" should be "auth0|64089494e980aedd96740212"
+        And the JSON response at "email" should be "dev+e2etest@publiq.be"
+
+    Scenario: Getting the creator of an ownership as a different user
+        Given I am authorized as JWT provider v2 user "dev_e2e_test"
+        And I create a minimal organizer and save the "id" as "organizerId"
+        And I wait for the organizer with url "/organizers/%{organizerId}" to be indexed
+        And I request ownership for "d759fd36-fb28-4fe3-8ec6-b4aaf990371d" on the organizer with organizerId "%{organizerId}" and save the "id" as "ownershipId"
+        And I approve the ownership with ownershipId "%{ownershipId}"
+
+        When I am authorized as JWT provider v2 user "invoerder"
+        And I send a GET request to "/ownerships/%{ownershipId}/creator"
+        And show me the unparsed response
```

Comment:
There is still a debug statement here & locally this test fails with me.

Reply:
I'll remove this line.
The feature tests of ownership all succeed on my computer with the latest changes from main

Reply:
To be sure I started the acceptance tests on [Jenkins](https://jenkins.publiq.be/job/uitdatabank-acceptance-tests/) for this feature branch

Reply:
All the tests are green